client/state/current-user/email-verification/reducer.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { createReducer } from 'state/utils';
+// import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	EMAIL_VERIFY_REQUEST,
+	EMAIL_VERIFY_REQUEST_SUCCESS,
+	EMAIL_VERIFY_REQUEST_FAILURE,
+	EMAIL_VERIFY_STATE_RESET,
+} from 'state/action-types';
+
+export default createReducer( {}, {
+	[ EMAIL_VERIFY_REQUEST ]: () => ( {
+		pendingRequest: true,
+		emailSent: false,
+		error: false,
```

Comment:
In general I like reducers to manage the simplest state possible. So we could instead do something like:
```jsx
const pendingRequest = createReducer( false, {
	[ EMAIL_VERIFY_REQUEST ]: stubTrue,
	[ EMAIL_VERIFY_REQUEST_SUCCESS ]: stubFalse,
	[ EMAIL_VERIFY_REQUEST_FAILURE ]: stubFalse,
} );
//...
export default combineReducers( {
    pendingRequest,
    emailSent,
    error,
} );
```

We likely won't need to keep track of pending state if we use the data-layer, but just wanted to point out the suggestion for other state trees.